tests/hikari/test_templates.py

```diff
@@ -19,25 +19,76 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+import mock
 import pytest
 
 from hikari import templates
+from hikari import users
 
 
 class TestTemplate:
     @pytest.fixture()
-    def obj(self):
+    def mock_user(self):
+        return mock.Mock(return_value=users.User)
```

Comment:
This is not technically correct, as we dont want it to return a type. You can just keep it as:
```suggestion
        return mock.Mock()
```
and inline it bellow

Reply:
I just used `mock_app` instead